cspell/.cspell.json

```diff
@@ -560,6 +560,8 @@
         "linuxcan",
         "LIVOX",
         "LLVM",
+        "Logiee_S-TC",
+        "Logiee_SS",
```

Comment:
Thanks! But the words are split by `_` and `-` when checking, and the words less than four characters are ignored. So `Logiee` would be the only one we need.
```suggestion
        "Logiee",
```

Reply:
Thanks for your review. I'll fix it

Reply:
FYI: Just clicking the commit suggestion button would save your time.
![image](https://user-images.githubusercontent.com/1256980/130179229-1d00a540-36dd-4192-967c-ae2ea4e7926d.png)